layers/ntp.go

```diff
@@ -0,0 +1,450 @@
+//******************************************************************************
+//
+// Copyright 2016 Ross N. Williams. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file in the root of the source
+// tree.
+//
+//******************************************************************************
+
+package layers
+
+import (
+	"encoding/binary"
+	"fmt"
+	"github.com/google/gopacket"
+)
+
+//******************************************************************************
+//
+// Network Time Protocol (NTP) Decoding Layer
+// ------------------------------------------
+// This file provides a GoPacket decoding layer for NTP.
+//
+// More specifically, this file adds to the package "layers" an additional
+// type called "NTP" that conforms to the interface type DecodingLayer
+// that is defined in the package:
+//
+//    src/github.com/google/gopacket/parser.go
+//
+// To do this, it must implement the following functions:
+//
+//     DecodeFromBytes(data []byte, df DecodeFeedback) error
+//     CanDecode() LayerClass
+//     NextLayerType() LayerType
+//     LayerPayload() []byte
+//
+//******************************************************************************
```

Comment:
I'd remove lines 25-36, as they're common knowledge.  Definitely keep the rest of the stuff, though, it's super useful.